mycroft/configuration/mycroft.conf

```diff
@@ -391,8 +391,14 @@
     // ('mycroft_mark_1', 'picroft', 'mycroft_mark_2pi')
     // to disable forced ntp_sync in official mycroft platforms
     // set this to false
-    "force_mycroft_ntp": true
+    "force_mycroft_ntp": true,
 
+    // Uncomment or add "idle_display_skill" to set initial homescreen
+    // "idle_display_skill": "skill-ovos-homescreen.openvoiceos",
+
+    // Extensions provide additional GUI platform support for specific devices
+    // Currently supported devices: SmartSpeaker, Bigscreen or Generic
+    "extension": "Generic"
```

Comment:
can this be moved out of enclosure into the gui section of mycroft.conf?

I'm hoping we can drop "enclosure" soon and never use that word again :P

Reply:
I would move it to ``` "GUI": { "extension": "Generic" } ``` but "idle_display_skill" will still remain in "enclosure" if it needs to keep compact with mycroft-core, I would just like to keep them together, either i move both to "GUI" or keep them both here

Reply:
ok will look into it

Reply:
Done